src/condor_starter.V6.1/starter.cpp

```diff
@@ -3543,6 +3547,23 @@ Starter::PublishToEnv( Env* proc_env )
 				}
 			}
 		}
+
+		const ClassAd * msec = jic->getMachineSecetsAd();
+		if (msec) {
+			// give the job access to the split claim id, (if there is one)
+			// we unparse so that the value will have "" and internal "" will be escaped
```

Comment:
Why is quoting here useful?

Reply:
If you want to have a script echo "claimid=$envname" then you need $envname to be classad quoted.

Reply:
And if the callee wants to do most other things (write to a file, use with the python bindings, etc), they need to unquote it. This feels like something intended for internal development/testing use, so I'll leave it alone.